cmd/crc/cmd/daemon.go

```diff
@@ -0,0 +1,35 @@
+package cmd
+
+import (
+	"github.com/code-ready/crc/pkg/crc/api"
+	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/code-ready/crc/pkg/crc/logging"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+func init() {
+	rootCmd.AddCommand(daemonCmd)
+}
+
+var daemonCmd = &cobra.Command{
+	Use:    "daemon",
+	Short:  "Run the crc daemon",
+	Long:   "Run the crc daemon",
+	Hidden: true,
+	Run: func(cmd *cobra.Command, args []string) {
+		runDaemon()
+	},
+}
+
+func runDaemon() {
+	//setup separte logfile for daemon
+	logging.CloseLogging()
```

Comment:
Yeah, but now we need it since, we set the log file in root.go which runs before this, so its needed to overwrite that and set it to the new daemon log file path.

Reply:
can't this behaviour be somehow fixed?